### PR TITLE
Deploy name tweaks

### PR DESCRIPTION
**Pull request by LindsayYoung**
takes care of , most of [finish cloud deploy](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/141)
Once we can confirm this works, we can set the default branch to develop.
## What does this change?

fixes a few naming inconsistencies so that we can deploy from circle to dev and staging

## Screenshots (for front-end PR):
successful deploy to Dev:
https://circleci.com/gh/usdoj-crt/crt-portal/51

successful deploy to Stage:
https://circleci.com/gh/usdoj-crt/crt-portal/56

Merging this to Master will confirm that works too

## Checklist:

_these will eventually be enforced by CircleCI; we need to set that up at the USDOJ org level_

+ [ ] If front end or functionality change, run locally and check http://0.0.0.0:8000/report/.
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests)

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -118,7 +118,7 @@ jobs:
       - run:
           name: Deploy to dev
           command: |
-              ./blue-green-deploy.sh $USERNAME_DEV $CF_PASSWORD_DEV dev
+              ./blue-green-deploy.sh $CRT_USERNAME_DEV $CRT_PASSWORD_DEV dev
```
Comment (alexsoble) [inline .circleci/config.yml]: Gotcha! So `$CRT_USERNAME_DEV` etc. all need to be set correctly in Circle for the deploy scripts to work properly? I think that would be helpful to note in a code comment.

File: .circleci/config.yml [hunk 2]
```diff
@@ -248,7 +248,7 @@ workflows:
             - build_and_test
           filters:
             branches:
-              only: release/*
+              only: /^release.*/
```
Comment (alexsoble) [inline .circleci/config.yml]: So this matches `release` and `release-1`, but rejects `1-release` and `ars-release` — does that summarize the change well?

Comment (LindsayYoung) [reply .circleci/config.yml]: yea, it is looking for branches starting with "release"

File: .circleci/config.yml (hunk 1 above)
Comment (LindsayYoung) [reply .circleci/config.yml]: Ok I added a comment to the top of the file and the instructions for adding the vars are already in the readme https://github.com/usdoj-crt/crt-portal#initial-cloudgov-set-up

**Pull request merged**